frontend/components/Documents.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from "react";
+// import * as PropTypes from "prop-types";
+// import STYLES from "./Documents.module.scss";
+
+const Documents = () => {
+
+    const [docData, setDocData] = useState(null);
```

Comment:
This set up overall works and makes sense, though I'll offer an opinionated opinion here:

One of the benefits of having to set the initial value in our `useState` hook here is that we can, effectively, type hint what the shape of the data will be that we expect the API to return to the frontend. In this case, `docData`, once data is returned and set, will be an array. We could, then, write `useState([])` and indicate right here at the top of the file that that's what we're hoping for. That will obviously break your pattern below, where you use the falseyness of the initial `null` value in line 24 as a check against a loading state. But I think it's worth being more explicit here by setting a second loading state (`const [loading, setLoading] = useState(true);`) and using that in line 24. It's certainly more verbose, but also more explicit.

Reply:
A new loading state has been added.

Reply:
:awesome: Did you have any thoughts about changing the `null` initial value to `[]` in `[docData, setDocData]`?